src/Riverline/MultiPartParser/Part.php

```diff
@@ -46,7 +46,7 @@ public function __construct($content)
     protected function parseContentString($content)
     {
         // Split headers and body
-        $splits = preg_split('/(\r?\n){2}/', $content, 2);
```

Comment:
Can you enforce them or at least ask for software updates? I suspect major versions of email servers/clients as well as major public email services all respect RFCs.
As you quoted `MAY recognize`, which means it is pretty valid to ignore those incorrect messages.
Anyway, if this is such a bit issue - I'll revert back to regular expressions:(